src/service.rs

```diff
@@ -403,9 +402,6 @@ where
 			relay_chain_interface,
 			relay_chain_slot_duration,
 			import_queue,
-			collator_options: CollatorOptions {
```

Comment:
the new API doesn't take these collator options pointing to the relay chain and I am testing spinning off a local environment and I am unable to onboard the parachain (for now). Will update if this persists